sql/logictest: TestParallel failed

sql/logictest.TestParallel [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9377227?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_StressBazel/9377227?buildTab=artifacts#/) on master @ [99102ddf4b7602788b422366f1acc14b81c64d24](https://github.com/cockroachdb/cockroach/commits/99102ddf4b7602788b422366f1acc14b81c64d24):


```
W230401 12:15:08.869579 57273009 sql/sqlstats/persistedsqlstats/flush.go:184  [T1,n1] 132640  failed to flush statement statistics: flush statement 14921957443287757346's statistics: insert-stmt-stats: node unavailable; try another peer
W230401 12:15:08.873449 57044520 sql/sqlstats/persistedsqlstats/flush.go:184  [T1,n2] 132641  failed to flush statement statistics: flush statement 8225077141659700325's statistics: insert-stmt-stats: node unavailable; try another peer
W230401 12:15:08.874121 57273009 sql/sqlstats/persistedsqlstats/flush.go:184  [T1,n1] 132642  failed to flush statement statistics: flush statement 177636696958847028's statistics: insert-stmt-stats: node unavailable; try another peer
W230401 12:15:08.878810 57273009 sql/sqlstats/persistedsqlstats/flush.go:184  [T1,n1] 132643  failed to flush statement statistics: flush statement 4077394443941908411's statistics: insert-stmt-stats: node unavailable; try another peer
W230401 12:15:08.883288 57273009 sql/sqlstats/persistedsqlstats/flush.go:184  [T1,n1] 132644  failed to flush statement statistics: flush statement 12819062270244084721's statistics: insert-stmt-stats: node unavailable; try another peer
W230401 12:15:08.887999 57273009 sql/sqlstats/persistedsqlstats/flush.go:184  [T1,n1] 132645  failed to flush statement statistics: flush statement 1879849972839045049's statistics: insert-stmt-stats: node unavailable; try another peer
W230401 12:15:08.890900 57273009 sql/sqlstats/persistedsqlstats/flush.go:184  [T1,n1] 132646  failed to flush statement statistics: flush statement 8931725617162097219's statistics: insert-stmt-stats: node unavailable; try another peer
W230401 12:15:08.893481 57273009 sql/sqlstats/persistedsqlstats/flush.go:184  [T1,n1] 132647  failed to flush statement statistics: flush statement 3193856872155801960's statistics: insert-stmt-stats: node unavailable; try another peer
W230401 12:15:08.896164 57273009 sql/sqlstats/persistedsqlstats/flush.go:184  [T1,n1] 132648  failed to flush statement statistics: flush statement 16956964122461726811's statistics: insert-stmt-stats: node unavailable; try another peer
W230401 12:15:08.898617 57273009 sql/sqlstats/persistedsqlstats/flush.go:184  [T1,n1] 132649  failed to flush statement statistics: flush statement 18196689053367598556's statistics: insert-stmt-stats: node unavailable; try another peer
W230401 12:15:08.900557 57273009 sql/sqlstats/persistedsqlstats/flush.go:184  [T1,n1] 132650  failed to flush statement statistics: flush statement 1087927460085071165's statistics: insert-stmt-stats: node unavailable; try another peer
W230401 12:15:08.902952 57273009 sql/sqlstats/persistedsqlstats/flush.go:184  [T1,n1] 132651  failed to flush statement statistics: flush statement 13953538446758315263's statistics: insert-stmt-stats: node unavailable; try another peer
W230401 12:15:08.906541 57273009 sql/sqlstats/persistedsqlstats/flush.go:184  [T1,n1] 132652  failed to flush statement statistics: flush statement 10636516999895996107's statistics: insert-stmt-stats: node unavailable; try another peer
W230401 12:15:08.909322 57273009 sql/sqlstats/persistedsqlstats/flush.go:184  [T1,n1] 132653  failed to flush statement statistics: flush statement 8594249169360073544's statistics: insert-stmt-stats: node unavailable; try another peer
W230401 12:15:08.911578 57273009 sql/sqlstats/persistedsqlstats/flush.go:184  [T1,n1] 132654  failed to flush statement statistics: flush statement 4553633081865308432's statistics: insert-stmt-stats: node unavailable; try another peer
W230401 12:15:08.913541 57273009 sql/sqlstats/persistedsqlstats/flush.go:184  [T1,n1] 132655  failed to flush statement statistics: flush statement 16037735350525453646's statistics: insert-stmt-stats: node unavailable; try another peer
W230401 12:15:08.915491 57273009 sql/sqlstats/persistedsqlstats/flush.go:184  [T1,n1] 132656  failed to flush statement statistics: flush statement 16037756241246389521's statistics: insert-stmt-stats: node unavailable; try another peer
W230401 12:15:08.917887 57273009 sql/sqlstats/persistedsqlstats/flush.go:184  [T1,n1] 132657  failed to flush statement statistics: flush statement 14315065430772624722's statistics: insert-stmt-stats: node unavailable; try another peer
W230401 12:15:08.920667 57273009 sql/sqlstats/persistedsqlstats/flush.go:184  [T1,n1] 132658  failed to flush statement statistics: flush statement 15117588807398990307's statistics: insert-stmt-stats: node unavailable; try another peer
W230401 12:15:08.923426 57273009 sql/sqlstats/persistedsqlstats/flush.go:184  [T1,n1] 132659  failed to flush statement statistics: flush statement 12938408181975031625's statistics: insert-stmt-stats: node unavailable; try another peer
W230401 12:15:08.926182 57273009 sql/sqlstats/persistedsqlstats/flush.go:184  [T1,n1] 132660  failed to flush statement statistics: flush statement 18445459025402006891's statistics: insert-stmt-stats: node unavailable; try another peer
W230401 12:15:08.928142 57273009 sql/sqlstats/persistedsqlstats/flush.go:184  [T1,n1] 132661  failed to flush statement statistics: flush statement 14208342361646805591's statistics: insert-stmt-stats: node unavailable; try another peer
W230401 12:15:08.930860 57273009 sql/sqlstats/persistedsqlstats/flush.go:184  [T1,n1] 132662  failed to flush statement statistics: flush statement 8376061731536268075's statistics: insert-stmt-stats: node unavailable; try another peer
W230401 12:15:08.933299 57273009 sql/sqlstats/persistedsqlstats/flush.go:184  [T1,n1] 132663  failed to flush statement statistics: flush statement 2724220997590187138's statistics: insert-stmt-stats: node unavailable; try another peer
W230401 12:15:08.936226 57273009 sql/sqlstats/persistedsqlstats/flush.go:184  [T1,n1] 132664  failed to flush statement statistics: flush statement 7380975359776190197's statistics: insert-stmt-stats: node unavailable; try another peer
W230401 12:15:08.939056 57273009 sql/sqlstats/persistedsqlstats/flush.go:184  [T1,n1] 132665  failed to flush statement statistics: flush statement 7380963265148279738's statistics: insert-stmt-stats: node unavailable; try another peer
W230401 12:15:08.941631 57273009 sql/sqlstats/persistedsqlstats/flush.go:184  [T1,n1] 132666  failed to flush statement statistics: flush statement 4407173395255379546's statistics: insert-stmt-stats: node unavailable; try another peer
W230401 12:15:08.943666 57273009 sql/sqlstats/persistedsqlstats/flush.go:184  [T1,n1] 132667  failed to flush statement statistics: flush statement 14490211381980301883's statistics: insert-stmt-stats: node unavailable; try another peer
W230401 12:15:08.946369 57273009 sql/sqlstats/persistedsqlstats/flush.go:184  [T1,n1] 132668  failed to flush statement statistics: flush statement 9344437996277039138's statistics: insert-stmt-stats: node unavailable; try another peer
W230401 12:15:08.948999 57273009 sql/sqlstats/persistedsqlstats/flush.go:184  [T1,n1] 132669  failed to flush statement statistics: flush statement 13770373304407341079's statistics: insert-stmt-stats: node unavailable; try another peer
W230401 12:15:08.951780 57273009 sql/sqlstats/persistedsqlstats/flush.go:184  [T1,n1] 132670  failed to flush statement statistics: flush statement 14957532621729412307's statistics: insert-stmt-stats: node unavailable; try another peer
W230401 12:15:08.954669 57273009 sql/sqlstats/persistedsqlstats/flush.go:184  [T1,n1] 132671  failed to flush statement statistics: flush statement 3927939360131033573's statistics: insert-stmt-stats: node unavailable; try another peer
W230401 12:15:08.957525 57273009 sql/sqlstats/persistedsqlstats/flush.go:184  [T1,n1] 132672  failed to flush statement statistics: flush statement 6219680929205217338's statistics: insert-stmt-stats: node unavailable; try another peer
W230401 12:15:08.960577 57273009 sql/sqlstats/persistedsqlstats/flush.go:184  [T1,n1] 132673  failed to flush statement statistics: flush statement 18438780628890563601's statistics: insert-stmt-stats: node unavailable; try another peer
W230401 12:15:08.963373 57273009 sql/sqlstats/persistedsqlstats/flush.go:184  [T1,n1] 132674  failed to flush statement statistics: flush statement 8493476170941479432's statistics: insert-stmt-stats: node unavailable; try another peer
W230401 12:15:08.966283 57273009 sql/sqlstats/persistedsqlstats/flush.go:184  [T1,n1] 132675  failed to flush statement statistics: flush statement 12292879387043088261's statistics: insert-stmt-stats: node unavailable; try another peer
W230401 12:15:08.969526 57273009 sql/sqlstats/persistedsqlstats/flush.go:184  [T1,n1] 132676  failed to flush statement statistics: flush statement 9099898713836500970's statistics: insert-stmt-stats: node unavailable; try another peer
W230401 12:15:08.972432 57273009 sql/sqlstats/persistedsqlstats/flush.go:184  [T1,n1] 132677  failed to flush statement statistics: flush statement 12215130198365885682's statistics: insert-stmt-stats: node unavailable; try another peer
W230401 12:15:08.975189 57273009 sql/sqlstats/persistedsqlstats/flush.go:184  [T1,n1] 132678  failed to flush statement statistics: flush statement 17033636905401300538's statistics: insert-stmt-stats: node unavailable; try another peer
W230401 12:15:08.977570 57273009 sql/sqlstats/persistedsqlstats/flush.go:184  [T1,n1] 132679  failed to flush statement statistics: flush statement 7376324857948889596's statistics: insert-stmt-stats: node unavailable; try another peer
W230401 12:15:08.980353 57273009 sql/sqlstats/persistedsqlstats/flush.go:184  [T1,n1] 132680  failed to flush statement statistics: flush statement 14589258048272698268's statistics: insert-stmt-stats: node unavailable; try another peer
W230401 12:15:08.983111 57273009 sql/sqlstats/persistedsqlstats/flush.go:184  [T1,n1] 132681  failed to flush statement statistics: flush statement 12740016567863122526's statistics: insert-stmt-stats: node unavailable; try another peer
W230401 12:15:08.986004 57273009 sql/sqlstats/persistedsqlstats/flush.go:184  [T1,n1] 132682  failed to flush statement statistics: flush statement 11336305007606786728's statistics: insert-stmt-stats: node unavailable; try another peer
W230401 12:15:08.988237 57273009 sql/sqlstats/persistedsqlstats/flush.go:184  [T1,n1] 132683  failed to flush statement statistics: flush statement 18097629588326401083's statistics: insert-stmt-stats: node unavailable; try another peer
W230401 12:15:08.990713 57273009 sql/sqlstats/persistedsqlstats/flush.go:184  [T1,n1] 132684  failed to flush statement statistics: flush statement 10735525030619903412's statistics: insert-stmt-stats: node unavailable; try another peer
W230401 12:15:08.993256 57273009 sql/sqlstats/persistedsqlstats/flush.go:184  [T1,n1] 132685  failed to flush statement statistics: flush statement 182084216068943938's statistics: insert-stmt-stats: node unavailable; try another peer
W230401 12:15:08.995663 57273009 sql/sqlstats/persistedsqlstats/flush.go:184  [T1,n1] 132686  failed to flush statement statistics: flush statement 6348203192952515288's statistics: insert-stmt-stats: node unavailable; try another peer
W230401 12:15:08.997668 57273009 sql/sqlstats/persistedsqlstats/flush.go:184  [T1,n1] 132687  failed to flush statement statistics: flush statement 8225077141659700325's statistics: insert-stmt-stats: node unavailable; try another peer
W230401 12:15:09.000059 57273009 sql/sqlstats/persistedsqlstats/flush.go:184  [T1,n1] 132688  failed to flush statement statistics: flush statement 1531212793301611594's statistics: insert-stmt-stats: node unavailable; try another peer
    --- FAIL: TestParallel/create_stats (2661.47s)
```
<p>Parameters: <code>TAGS=bazel,gss,deadlock</code>
</p>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
/cc @cockroachdb/sql-queries
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestParallel.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>


Jira issue: CRDB-26407

--- a/pkg/cmd/roachtest/tests/tpcc.go
+++ b/pkg/cmd/roachtest/tests/tpcc.go
@@ -93,11 +93,6 @@ type tpccOptions struct {
 	// also be doing a rolling-restart into the new binary while the cluster
 	// is running, but that feels like jamming too much into the tpcc setup.
 	Start func(context.Context, test.Test, cluster.Cluster)
-	// EnableCircuitBreakers causes the kv.replica_circuit_breaker.slow_replication_threshold
-	// setting to be populated, which enables per-Replica circuit breakers.
-	//
-	// TODO(tbg): remove this once https://github.com/cockroachdb/cockroach/issues/74705 is completed.
-	EnableCircuitBreakers bool
 	// SkipPostRunCheck, if set, skips post TPC-C run checks.
 	SkipPostRunCheck              bool
 	DisableDefaultScheduledBackup bool
@@ -165,10 +160,6 @@ func setupTPCC(
 		opts.Start(ctx, t, c)
 		db := c.Conn(ctx, t.L(), 1)
 		defer db.Close()
-		if opts.EnableCircuitBreakers {
-			_, err := db.Exec(`SET CLUSTER SETTING kv.replica_circuit_breaker.slow_replication_threshold = '15s'`)
-			require.NoError(t, err)
-		}
 
 		if t.SkipInit() {
 			return
@@ -508,10 +499,9 @@ func registerTPCC(r registry.Registry) {
 			headroomWarehouses := int(float64(maxWarehouses) * 0.7)
 			t.L().Printf("computed headroom warehouses of %d\n", headroomWarehouses)
 			runTPCC(ctx, t, c, tpccOptions{
-				Warehouses:            headroomWarehouses,
-				Duration:              120 * time.Minute,
-				SetupType:             usingImport,
-				EnableCircuitBreakers: true,
+				Warehouses: headroomWarehouses,
+				Duration:   120 * time.Minute,
+				SetupType:  usingImport,
 			})
 		},
 	})

--- a/pkg/sql/logictest/parallel_test.go
+++ b/pkg/sql/logictest/parallel_test.go
@@ -218,6 +218,11 @@ func (t *parallelTest) setup(ctx context.Context, spec *parTestSpec) {
 		r0.Exec(t, `UPDATE system.zones SET config = $2 WHERE id = $1`, objID, buf)
 	}
 
+	// Disable the circuit breakers on this cluster because they can lead to
+	// rare test flakes since the machine is likely to be overloaded when
+	// running TestParallel.
+	r0.Exec(t, `SET CLUSTER SETTING kv.replica_circuit_breaker.slow_replication_threshold = '0s'`)
+
 	if testing.Verbose() || log.V(1) {
 		log.Infof(t.ctx, "Creating database")
 	}

